### ISSUE TITLE
App names overlap again

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#27536]
![Bildschirmfoto von 2021-06-17 09-57-35](https://user-images.githubusercontent.com/213943/122355874-8d4f8980-cf52-11eb-84a7-4edc5b5bca15.png)

### Steps

* Have an app with a long name (or translation)
* If the app is active or hovered it will overlap the next entry causing readability issues (depending on language/font/appname)

cc @jancborchardt

**Comment by szaimen** [comment]
It is like this since https://github.com/nextcloud/server/pull/22924

**Comment by szaimen** [comment]
I'll just leave this here which fixes this if the appmenu doesn't have a gradient but I am not sure if it is the best fix to this problem:
```scss
#appmenu:hover li span {
    background-color: var(--color-primary);
    z-index: 3;
    border-radius: 10px;
}
#appmenu:hover li svg {
    z-index: 4;
}
```

**Comment by szaimen** [comment]
cc @nextcloud/designers

**Comment by einwickler** [comment]
The appnames don't even need to be that long for this to occur.
I even have some overlap with Contacts and Calendar:
![grafik](https://user-images.githubusercontent.com/29920253/211029298-e01385a7-cf55-4454-936e-0e0e07c0ef93.png)

**Comment by jancborchardt** [comment]
So @JuliaKirschenheuter worked on this as part of accessibility improvements and app names do not overlap anymore.

It will be released with Nextcloud 26, correct @JuliaKirschenheuter? (And do you maybe have a screenshot to illustrate it?)

**Comment by nickvergessen** [comment]
![image](https://user-images.githubusercontent.com/213943/211279664-b72681f7-3311-4059-a259-7f54ecf93c31.png)

yeah not happening anymore in master

**Comment by szaimen** [comment]
🎉
